db/migrations/mysql/000105_remove_tokens.up.sql

```diff
@@ -0,0 +1,18 @@
+CREATE PROCEDURE Remove_Tokens_If_Exist ()
+BEGIN
+    DECLARE C INT;
+
+    SELECT COUNT(o.`Token`) INTO C from OAuthAccessData o 
+    LEFT JOIN Preferences p ON o.clientid = p.name AND o.userid = p.userid AND p.category = 'oauth_app'
+    INNER JOIN Sessions s ON o.token = s.token
+    WHERE p.name IS NULL;
+
+    IF(C > 0) THEN
+        DELETE o, s from OAuthAccessData o 
+        LEFT JOIN Preferences p ON o.clientid = p.name AND o.userid = p.userid AND p.category = 'oauth_app'
+        INNER JOIN Sessions s ON o.token = s.token
+        WHERE p.name IS NULL;
```

Comment:
Here `p.name is NULL` is included, but in the original PR, it isn't. Could you clarify?

Also sorry I missed https://github.com/mattermost/mattermost-server/pull/23036/files#r1185672462